__tests__/index.tsx

```diff
@@ -582,5 +604,36 @@ describe('VideoRenderer', () => {
       expect(onError).toHaveBeenCalledWith(e);
       expect(component.state().error).toEqual('some-error');
     });
+
+    it('should notify every other second when play time changes', () => {
+      const onTimeChange = jest.fn<
+        ReturnType<Required<VideoProps>['onTimeChange']>,
+        Parameters<Required<VideoProps>['onTimeChange']>
+      >();
+      const { component } = setup({
+        onTimeChange,
+      });
+
+      simulate(component, 'timeUpdate', {
+        currentTime: 10.5,
+        buffered: {}
+      });
+      simulate(component, 'timeUpdate', {
+        currentTime: 10.6,
+        buffered: {}
+      });
+      simulate(component, 'timeUpdate', {
+        currentTime: 11.2,
+        buffered: {}
+      });
+      simulate(component, 'timeUpdate', {
+        currentTime: 11.5,
+        buffered: {}
+      });
+
+      expect(onTimeChange).toHaveBeenCalledTimes(2);
+      expect(onTimeChange).toHaveBeenCalledWith(10);
+      expect(onTimeChange).toHaveBeenCalledWith(11);
```

Comment:
Should we assert the number of times onTimeChange gets called?

Reply:
we should. Which I do 2 lines above :-P